test/e2e/internal/utils/match/status.go

```diff
@@ -77,6 +77,9 @@ func newStateMachine(cmd string) *stateMachine {
 		sm.addPath("Skipped")
 		sm.addPath("Exists")
 		sm.addPath("Mounted")
+	case "manifest", "blob": // for `manifest push` and `blob push`
```

Comment:
What if we are adding status tests for `oras manifest delete` or `oras manifest blob delete`?

Looks like we need to improve `NewStatusMatcher` to take the whole argument and do a best-effort prefix matching.

Reply:
We can open an issue to track this.

Reply:
Put a TODO in the code and opened an issue: https://github.com/oras-project/oras/issues/1571